apollo-configservice/src/main/java/com/ctrip/framework/apollo/configservice/wrapper/DeferredResultWrapper.java

```diff
@@ -64,4 +65,10 @@ public void setResult(List<ApolloConfigNotification> notifications) {
   public DeferredResult<ResponseEntity<List<ApolloConfigNotification>>> getResult() {
     return result;
   }
+
+  @Override
+  public int compareTo(@NonNull DeferredResultWrapper deferredResultWrapper) {
+    int result = this.hashCode() - deferredResultWrapper.hashCode();
```

Comment:
How about just  `return Integer.compare(this.hashCode(), deferredResultWrapper.hashCode())`?